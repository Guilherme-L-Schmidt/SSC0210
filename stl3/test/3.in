200
cc
cdaabaacbdacababdd
bbbabbabcbbdbacbcbaadadca
ddbbaccadcabaca
cbddadcabadaacbdbcaaacadaddb
cdc
aaacccca
bbcdabcacdaabbbddc
abaadc
adbacddbbacdbcdddccbbbdaaaaccb
dbaddcbdbb
dcadbcad
dadabbcdddcdcadcabdabbaacb
bddadad
ddccaacbdcbbcc
abcabddbaacdddaaacada
abadbccdacaaaab
acdddccadbdcdbabdd
aaabbcacacdca
ccbdaabbccbbbcbcbcbaddcacacadbba
cdab
abdabbddadcdccaadaa
dbbbbacccaadcdccbccacba
c
adaacdbccacacbca
dcbdbbddaaabbdcbabdaabcdcbca
adcdbddcaaacdbababcccbca
aadcbacdabbbdbcbaddca
badbaabcbabdacbbacdacdca
acbbab
caaaaaabcccadaddbc
bacdcabdbcd
dadbbbcaaadaaabccabdbbccbdbcadac
abdacadbaababbbdabcadbadaacdabcd
ccbbadaddadcbbcdaaaabcda
bbb
accbaacaddbdbaadcddaac
acaabbccadcdcdbddb
ddcd
bcbbad
bcca
aaaadaaccdaba
badbaaacaddcd
bbcdabccdddacabcbdbca
adaaacddccbbadacbbdab
c
cbcadcadddabacdaccdabddbdb
dcbbabb
dbdbabccadacaddd
abdcdddabdcacdcdbbcbb
cbdcdcbcbaabbabcaaaabdcccbada
cadcadbd
acbbaabadabcbbbbcabbbc
c
abcbaadadbbcbbaabd
bcdaaacdaaccadbcbbbcabccaabda
bb
acbc
ccdacadabcabacbccccbba
aaacdccdaadbcbcbbbbaccccbab
bdbabcbcacccbacdcacadcc
dbaabcddbdad
cdbdbbaadcdbabbbcdcbabcbdada
bdcdbbdbcdaabbccaccbdcadcdcaa
acdaadadccdbabddabadcdd
dcacdadacbbcbccacddbada
baddcbbadcdadbaacadcabacbb
ddabcbcaccbbddcccbbacacccc
accbabdcbc
ddbdadabbd
dbcacdbdcadaca
cdadbccdcabcbdcdccadcdadabcbd
cbccabcbbdbaddacd
caacacdcbc
acd
adcbccacddacbadbdacbdcd
bcacadabbcdbdacaccbccddcdb
b
dbdbbadd
cdaaaadccdbd
bcaadccaabbbbccbacbbddcbcaccd
a
cbbdad
bddacbdbcbdcaacdbcadcccca
adbdbbcaadbbabddccdddacaa
ababddccdcdbdbadcadaacbbccca
abcbcbaabdbbbdddcbbdabddcaddcabc
aaaadadbaac
dbddabbadddbacdcdcbcacbc
adddddbabbdbbbcddbddbbbccaa
c
bdddaddca
cacabdbbacddb
bab
dccbdacababbbbaabdbadcddadbbd
ddabdcbcaba
ccabddbacbcdbcaccdcbcd
aa
accbadadacbbdcacbbdc
bcabdcadcbbabadcbad
cbadbdabababbbdbcdadbccb
aacaadcacbccbbabccdddcc
bbbcdcdbcaddadaba
dcaccadbbbcddd
dac
cddccbaabccaabdcccacdcbad
cbcdacccaac
bbdccdddadb
abb
daccdacbdacccbc
cabbaab
bddccbaaddaadbcbdbcccb
aaacbda
bbbbbdbcbbccdaabbabdcdbcbddc
aaacdaaabacdcadacdddadbbacac
bdbacacbdacdcbbccbcbbccaddc
acabbbccdabcbcdccbdddcbdcbc
bcbbda
cbacddbbacdacabacaabcddcaaaa
acacccdca
cddbdaaddd
bbdb
bbcbccccaabaadc
caadca
ccbccbacc
b
cbbcdabdacaaaadcabdabcabaada
baaabaccccbddacbcaa
adabaaabb
abdcdcdcbcbdaabba
bcadcdcaacaabaaacdaddddbc
adca
accdcabbcaaacbabadadcdcc
dcdcabdaccadbaac
abaaa
baccbccdaacdcbdc
aacdcdbacacadccaaabacbd
bdbccaadbddaddaabcadbdbcbda
dcdbadadccdbaaacd
cbddcdbbcacaacbccdbcdbccccaa
acdcbdd
bcb
cddaabcdbcccabddbdcb
bbaaacdbdbdddccdbada
babdcbccabdbaaacaccdbdcbdacadb
cdbbdcaccacccacba
adbacbdcdaabbdcbda
abaabdbbdacccbabc
dcbadbbaaccbcaccb
bddccbbc
cbaacbaddbddcccbacddaaddbad
ccdacbcbabddcacaccddcdac
cadcabcababbbda
dbccbadcbbdbdbbdcacb
acaacbbbaddbcabdb
dbbdccdcca
bccacbcdcbadaacbcaccbaacddc
aaca
abdbdadabdadcbcaabbccccbbabd
bbcbcaaccbadadcacacbaddcc
bdcbbdbcdbada
dcbcaddcacaddddcbaaa
dabdabddcccdcabcbddcdbaaa
cdcabbbbcbddcccaacad
acddcccdadadbcdbb
ccbdbabbbaccbdadcdbbcaaccbcddcb
daddbccdbdddbdddddc
ddd
da
badbcddabcbcbaac
aacccadbddddbdccdddcbbab
c
c
acbb
baadcbdbddbdbb
aacd
adabdddbcaabadcaca
bbacdadbacdccadcaadcb
bddbacaabddadaccddaabcdcabc
dddbddbdacbcbddcaaaad
bcabd
adbbcaacddd
cacabcddcbcaabbacbaadcadcabacac
dbccdcacbaac
cbddb
bbc
cdcdbabbacabaadadbcbcbabbaa
ca
bdbcbcbb
dddadccddbdbabdaabdaabdbadacdacc
adbaacdccccacaacddadbccabbaaddbd
ccabbcadbcccbabbbcaaabbbbccbb
ccdcdbdcbcdac
ac
abb
abbabaabadbdaabcabb
ddcacbbddddbacbd
bbcddbdbaacc
cccac
dddcbcadabcaacbcdddadb
