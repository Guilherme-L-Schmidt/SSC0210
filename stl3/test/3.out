OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
c1
OK
OK
OK
OK
OK
OK
OK
c2
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
c3
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
b1
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
c4
c5
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
abb1
OK
OK
OK
OK
OK
